test/quadratic.jl

```diff
@@ -49,7 +49,7 @@ xlo, xhi = itp3[.2], itp3[xmax+.7]
 @test_approx_eq xhi A[end]
 
 # Check continuity
-xs = [0:.1:length(A)+1]
+xs = [0:.1:length(A)+1;]
```

Comment:
Is the `;` an idiom for something here (like getting a row vector instead of a col vector)? Why do we need that? As far as I can tell we're only using `xs` one value at a time, so dimensionality shouldn't matter as long as linear indexing works.